students/eowyn/session03/rot13.py

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python
+
+def rot13(a_string):
+    ''' Encode any string using rot13 encoding, preserving 
+    whitespace, punctuation, and capitalization '''
+    import codecs
+    return codecs.encode(a_string,encoding="rot13")
+
```

Comment:
Nice catch. There are other ways to do this, and not everyone notices there is a rot13 codec. :)